app/code/Magento/CatalogImportExport/Model/Import/Product/Option.php

```diff
@@ -1794,7 +1794,8 @@ protected function _getSpecificTypeData(array $rowData, $optionTypeId, $defaultS
             ];
 
             $priceData = false;
-            if (!empty($rowData[self::COLUMN_ROW_PRICE])) {
+            $customOptionRowPrice = $rowData[self::COLUMN_ROW_PRICE];
+            if (!empty($customOptionRowPrice) || $customOptionRowPrice === '0') {
```

Comment:
Hey @amatanovic. What if `customOPtionRowPrice` would be `'0.0'` for example?

Reply:
Hey @slavvka for '0.0' the empty() would return false so it worked even before this. The issue was with '0' because the empty() returns true so the price was never set.